doc/connectivity/bluetooth/api/audio/bluetooth-le-audio-arch.rst

```diff
@@ -59,6 +59,292 @@ GAF has been implemented in Zephyr with the following structure.
 
    Zephyr Generic Audio Framework
 
+
+Dependency on GATT
+==================
+
+The following tables show the required sub-procedures from GATT for each of the
+LE Audio profiles and services.
+
+.. table:: GATT sub-procedures for LE Audio profiles
+   :widths: auto
+
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+   | GATT sub-procedure                       | BAP | CCP | CAP | CSIP | GMAP | HAP | MCP | MICP | PBP | TMAP |
+   +==========================================+=====+=====+=====+======+======+=====+=====+======+=====+======+
+   | Exchange MTU                             | M   |     | I   |      |      | I   |     |      |     |      |
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
```

Comment:
Hmm, neither the BAP broadcast source or the CAP initiator for broadcast mandate the MTU exchange, so isn't this conditional? Same goes for any other GATT operation. 

These requirements are really per profile role, and not per profile, right?

Reply:
Hmm, yes, I'll make them conditional